commit 6e02e89f65ca1ca1d6ce30fbc557563164dd327e
Author: Eric Fiselier <eric@efcs.ca>
Date:   Thu Aug 3 04:28:10 2017 +0000

    Fix libcxx build with glibc 2.26+ by removing xlocale.h include.
    
    Patch by Khem Raj. Reviewed as D35697. Also see PR33729.
    
    
    
    git-svn-id: https://llvm.org/svn/llvm-project/libcxx/trunk@309920 91177308-0d34-0410-b5e6-96231b3b80d8

diff --git a/include/__locale b/include/__locale
index 766842294..91ed9e709 100644
--- a/include/__locale
+++ b/include/__locale
@@ -34,7 +34,7 @@
 # include <support/solaris/xlocale.h>
 #elif defined(_NEWLIB_VERSION)
 # include <support/newlib/xlocale.h>
-#elif (defined(__GLIBC__) || defined(__APPLE__)      || defined(__FreeBSD__) \
+#elif (defined(__APPLE__)      || defined(__FreeBSD__) \
     || defined(__EMSCRIPTEN__) || defined(__IBMCPP__))
 # include <xlocale.h>
 #elif defined(__Fuchsia__)
